.circleci/config.yml

```diff
@@ -176,13 +176,13 @@ jobs:
           command: "sudo apt-get update"
       - libvips
 
-      - install_solidus: { flags: "--sample=false --frontend=none --authentication=none" }
+      - install_solidus: { flags: "--sample=false --frontend=none --no-with-authentication" }
```

Comment:
nit:
Should this be `--with-authentication=false`?
I know it works both ways, but my guess is this "spelling" is more common.

Reply:
Okay, let's go with --with-authentication=false :+1: